contracts/adapters/AaveV3LeverageStrategyExtension.sol

```diff
@@ -66,4 +106,116 @@ contract AaveV3LeverageStrategyExtension is AaveLeverageStrategyExtension {
             abi.encodeWithSignature("setEModeCategory(address,uint8)", address(strategy.setToken), _categoryId);
         invokeManager(address(strategy.leverageModule), setEmodeCallData);
     }
+
+    function _calculateMaxBorrowCollateral(ActionInfo memory _actionInfo, bool _isLever) internal override view returns(uint256) {
+        
+        (uint256 maxLtvRaw, uint256 liquidationThresholdRaw) = _getLtvAndLiquidationThreshold();
+
+        // Normalize LTV and liquidation threshold to precise units. LTV is measured in 4 decimals in Aave which is why we must multiply by 1e14
+        // for example ETH has an LTV value of 8000 which represents 80%
+        if (_isLever) {
+            uint256 netBorrowLimit = _actionInfo.collateralValue
+                .preciseMul(maxLtvRaw.mul(10 ** 14))
+                .preciseMul(PreciseUnitMath.preciseUnit().sub(execution.unutilizedLeveragePercentage));
+
+            return netBorrowLimit
+                .sub(_actionInfo.borrowValue)
+                .preciseDiv(_actionInfo.collateralPrice);
+        } else {
+            uint256 netRepayLimit = _actionInfo.collateralValue
```

Comment:
👀 update to loosen restriction of `netRepayLimit` https://github.com/sherlock-audit/2023-05-Index-judging/issues/254